weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionsPage.tsx

```diff
@@ -332,7 +335,9 @@ const ObjectVersionsTable: React.FC<{
   // Highlight table row if it matches peek drawer.
   const query = useURLSearchParamsDict();
   const {peekPath} = query;
-  const peekId = peekPath ? peekPath.split('/').pop() : null;
+
+  const peekId = parseObjectVersionUrlPathToRefUri(peekPath);
```

Comment:
we might want to name this to reflect that it is the URI, or a comment saying that peekId == rowId == objectURI